config/krakend/scripts/fake_frinx_token.lua

```diff
@@ -0,0 +1,12 @@
+function fake_token_validate(request)
+    -- local header = request:headers('Fake-Frinx-Token')
+    -- if header ~= '' then
+        print("Fake Frinx Token used")
```

Comment:
maybe a different msg would be better. Users might not know what Fake Frinx Token really means, what if instead this said: "WARNING: Using default, hardcoded credentials" ... or similar, "WARNING" is not necessary maybe